src/astro/utils/path.py

```diff
@@ -111,3 +112,54 @@ def is_local(path):
     """
     file_location = get_location(path)
     return file_location == FileLocation.LOCAL
+
+
+def get_module_dot_notation(module_path: pathlib.Path) -> str:
+    """
```

Comment:
What is this function for?

Reply:
It converts `Path("/home/tati/Code/astro/src/astro/databases/sqlite.py")` to `"astro.databases.sqlite"`, and it is used by `get_database_from_conn_id` to automatically map connection types which have the same name as our modules to the respective file.